Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -1712,7 +1712,7 @@
     <value>For example, ‘You need to change the email address. The domain name in the organisation email address you gave us does not match the organisation name you want to register’.</value>
   </data>
   <data name="Supplementary_CookieDetails_TableRow_Preferences" xml:space="preserve">
-    <value />
+    <value>Stores the cookies accept / reject preference.</value>
```

Comment:
This key was missing a value. It had been duplicated initially, and then has been through many many merge conflicts in this file and it had got lost as a result